scilpy/tracking/seed.py

```diff
@@ -57,23 +59,32 @@ def get_next_pos(self, random_generator, indices, which_seed):
         seed_pos: tuple
             Position of next seed expressed in mm.
         """
-        len_seeds = len(self.seeds)
+        len_seeds = len(self.seeds_vox)
         if len_seeds == 0:
             return []
 
-        voxel_dim = np.asarray(self.voxres)
-
         # Voxel selection from the seeding mask
         ind = which_seed % len_seeds
-        x, y, z = self.seeds[indices[ind]]
-
-        # Subvoxel initial positioning
-        r_x = random_generator.uniform(0, voxel_dim[0])
-        r_y = random_generator.uniform(0, voxel_dim[1])
-        r_z = random_generator.uniform(0, voxel_dim[2])
-
-        return x * self.voxres[0] + r_x, y * self.voxres[1] \
-            + r_y, z * self.voxres[2] + r_z
+        x, y, z = self.seeds_vox[indices[ind]]
+
+        # Subvoxel initial positioning. Right now x, y, z are in vox space,
+        # origin=corner, so between 0 and 1.
+        r_x = random_generator.uniform(0, 1)
+        r_y = random_generator.uniform(0, 1)
+        r_z = random_generator.uniform(0, 1)
+
+        # Moving inside the voxel
+        x += r_x
+        y += r_y
+        z += r_z
+
+        assert self.origin == 'corner'
```

Comment:
Unlikely it would change since it is hardcoded. Maybe just make it extremely clear that the seed generator always returns seeds with origin corner? And remove the attribute `self.origin`?

Reply:
I added a line to support corner instead of forcing it. Looks ok?